### PR TITLE
reason-react-native: add AccessibilityInfo API

### PR DESCRIPTION
**Pull request by sgny**
I noticed that these were missing.

### CODE REVIEW

File: reason-react-native/src/apis/AccessibilityInfo.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+id: apis/AccesibilityInfo
+title: AccesibilityInfo
+wip: true
+---
+
+```reason
+type payload = {
```
Comment (cknitt) [inline reason-react-native/src/apis/AccessibilityInfo.md]: Shouldn't this be called `announcementFinishedPayload` (or maybe `announcementFinishedParams` to match other event types)?

Comment (sgny) [reply reason-react-native/src/apis/AccessibilityInfo.md]: How about announcementResult or announcementOutcome?

File: reason-react-native/src/apis/AccessibilityInfo.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  .
+  "announcement": string,
+  "success": bool,
+};
+
+[@bs.scope "AccesibilityInfo"] [@bs.module "react-native"]
```
Comment (MoOx) [inline reason-react-native/src/apis/AccessibilityInfo.md]: AccesibilityInfo => AccessibilityInfo

Should be adjusted everywhere :)

File: reason-react-native/src/apis/AccessibilityInfo.md (hunk 1 above)
Comment (MoOx) [reply reason-react-native/src/apis/AccessibilityInfo.md]: `announcementFinishedResult` seems a good compromise.
Not a fan of "params" that could be interpreted as an argument from the user.

Comment (cknitt) [reply reason-react-native/src/apis/AccessibilityInfo.md]: 👍

**Comment by MoOx** [comment]
@sgny You missed a place, and I cannot edit since you did that on your fork (reminder: you have write access on this repo, no need for a fork ^^).

Btw, maybe `announcementFinishedResult` would be more explicit that `*Paypload` ? (I am saying that since a payload look more that something that I would "give" than something I "receive" (but I am not English fluent so I can be wrong)

**Comment by sgny** [comment]
@MoOx I’ll fix that tomorrow, sorry about that.

Regarding payload you are more accurate, but I couldn’t find a more appropriate name at the time.

Would announcementResult be clear enough?

**Comment by MoOx** [comment]
I guess announcementResult is good enough

**Pull request merged**